fontforge/http.c

```diff
@@ -663,7 +663,7 @@ return( true );
 FILE *URLToTempFile(char *url,void *_lock) {
     FILE *ret;
 
-    if ( strncasecmp(url,"http://",7)==0 )
+    if ( strncasecmp(url,"http://",7)==0 || strncasecmp(url,"https://",8)==0 )
```

Comment:
You are right, I'll fix it, but it really needs double validation?

I'm new here... How could I run tests? I don't know how could I test it